src/components/Pressable/GenericPressable/PropTypes.js

```diff
@@ -0,0 +1,118 @@
+import PropTypes from 'prop-types';
+import stylePropType from '../../../styles/stylePropTypes';
+import CONST from '../../../CONST';
+
+const stylePropTypeWithFunction = PropTypes.oneOfType([
+    stylePropType,
+    PropTypes.func,
+]);
+
+const pressablePropTypes = {
+    /**
+     * onPress callback
+     */
+    onPress: PropTypes.func.isRequired,
+
+    /**
+     * Specifies keyboard shortcut to trigger onPressHandler
+     * @example {shortcutKey: 'a', modifiers: ['ctrl', 'shift'], descriptionKey: 'keyboardShortcut.description'}
+     */
+    keyboardShortcut: PropTypes.shape({
+        descriptionKey: PropTypes.string.isRequired,
+        shortcutKey: PropTypes.string.isRequired,
+        modifiers: PropTypes.arrayOf(PropTypes.string),
+    }),
+
+    /**
+     * Specifies if haptic feedback should be used on press
+     * @default false
+     */
+    shouldUseHapticsOnPress: PropTypes.bool,
+
+    /**
+     * Specifies if haptic feedback should be used on long press
+     * @default false
+     */
+    shouldUseHapticsOnLongPress: PropTypes.bool,
+
+    /**
+     * style for when the component is disabled. Can be a function that receives the component's state (active, disabled, hover, focus, pressed, isScreenReaderActive)
+     * @default {}
+     * @example {backgroundColor: 'red'}
+     * @example state => ({backgroundColor: state.active ? 'red' : 'blue'})
+     */
+    disabledStyle: stylePropTypeWithFunction,
+
+    /**
+     * style for when the component is hovered. Can be a function that receives the component's state (active, disabled, hover, focus, pressed, isScreenReaderActive)
+     * @default {}
+     * @example {backgroundColor: 'red'}
+     * @example state => ({backgroundColor: state.active ? 'red' : 'blue'})
```

Comment:
NAB but it might be a _slightly_ clearer example to change the `hoverStyle` based off `state.hovered` instead of `state.active` (`state => ({backgroundColor: state.hovered ? 'red' : 'blue'})`). Feedback applies to other props where the example is copy pasta as well.